Description: add missing #include directive for the use of std::function
Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1626972
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/image/imgFrame.h
+++ b/image/imgFrame.h
@@ -7,6 +7,7 @@
 #ifndef mozilla_image_imgFrame_h
 #define mozilla_image_imgFrame_h
 
+#include <functional>
 #include <utility>
 
 #include "AnimationParams.h"
